components/script/dom/document.rs

```diff
@@ -1265,6 +1268,56 @@ impl Document {
             .reflow(ReflowGoal::Full, ReflowReason::MouseEvent);
     }
 
+    #[allow(unsafe_code)]
+    pub fn handle_wheel_event(
+        &self,
+        js_runtime: *mut JSRuntime,
+        delta: WheelDelta,
+        client_point: Point2D<f32>,
+        node_Address: Option<UntrustedNodeAddress>,
+    ) {
+        let wheel_event_type_string = "wheel".to_owned();
```

Comment:
Is it necessary to have a String here? Doesn't `DOMString::new` work with a str?

Reply:
nope, `DOMString::new()` only creates an empty domstring. I could do `DOMString::new(); DOMString::push_str(&mut self, &str)`. I used `from_string` cause that's what the mouse event used.